OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/preferencecenter/OptimovePreferenceCenter.java

```diff
@@ -0,0 +1,293 @@
+package com.optimove.android.preferencecenter;
```

Comment:
should also bump sdk version (check readme checkboxes)

this change is not breaking (hopefully), so, should be minor version update 7.3.1 --> 7.4.0